--- GD-Convert-2.12/Convert.pm.orig	2005-05-12 22:37:59.000000000 +0200
+++ GD-Convert-2.12/Convert.pm	2006-06-23 09:48:03.000000000 +0200
@@ -34,11 +34,7 @@
 		} elsif ($GD::VERSION >= 2.15) {
 		    # hmmm ...
 		} elsif ($GD::VERSION >= 1.40 && !$installed{$f} && GD::Image->can($f)) {
-		    $@ = "";
-		    eval { GD::Image->new->$f(); };
-		    if ($@ !~ /libgd was not built with gif support/) {
-			undef $as;
-		    }
+		    undef $as;
 		}
 		# No? Then try alternatives
 		if (defined $as) {
